dis_snek/models/discord_objects/voice_state.py

```diff
@@ -0,0 +1,58 @@
+from typing import TYPE_CHECKING, Optional, Dict, Any
+
+from attr.converters import optional as optional_c
+
+from dis_snek.const import MISSING
+from dis_snek.models.discord import ClientObject
+from dis_snek.models.snowflake import to_snowflake
+from dis_snek.models.timestamp import Timestamp
+from dis_snek.utils.attr_utils import define, field
+from dis_snek.utils.converters import timestamp_converter
+
+if TYPE_CHECKING:
+    from dis_snek.client import Snake
+    from dis_snek.models import Guild, TYPE_VOICE_CHANNEL
+    from dis_snek.models.discord_objects.user import Member
+    from dis_snek.models.snowflake import Snowflake_Type
+
+
+@define()
+class VoiceState(ClientObject):
+    user_id: "Snowflake_Type" = field(default=MISSING, converter=to_snowflake)
+    session_id: str = field(default=MISSING)
+    deaf: bool = field(default=False)
+    mute: bool = field(default=False)
+    self_deaf: bool = field(default=False)
+    self_mute: bool = field(default=False)
+    self_stream: Optional[bool] = field(default=None)
+    self_video: bool = field(default=MISSING)
+    suppress: bool = field(default=MISSING)
```

Comment:
IMO type hinting should include MISSING as union with bool or sane bool defaults should be assumed

Reply:
Good catch, missed that!